src/test/java/edu/harvard/iq/dataverse/sitemap/SiteMapUtilTest.java

```diff
@@ -82,7 +82,8 @@ public void testUpdateSiteMap() throws IOException, ParseException {
         String tmpDir = tmpDirPath.toString();
         File docroot = new File(tmpDir + File.separator + "docroot");
         docroot.mkdirs();
-        System.setProperty("com.sun.aas.instanceRoot", tmpDir);
+        // TODO: this and the above should be replaced with JUnit 5 @TestDir
```

Comment:
Does this relate to any issue that you could link to?

Reply:
With updating this PR to latest develop, using JUnit5 constructs is possible. The refactored test resolves your comment.